data/aeejaz.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Ejaj Ahmed",
+  "type": "personal",
+  "bio": "AI, FUll-Stack , & Web3",
+
+  "tags": ["Full-stack", "Javascript", "Typescript", "AI", "Web3"],
+  "socials": [
+    {
+      "name": "Follow me on Twitter",
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/aeejazkhan"
+    },
+    {
+      "name": "Follow me on Github",
+      "icon": "FaGithub",
+      "url": "https://github.com/aeejaz"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "icon": "FaLinkedin",
+      "url": "https://youtube.com/aeejazkhan"
```

Please fix this broken URL as suggested. 

```suggestion
      "url": "https://youtube.com/@aeejazkhan"
```

---

Made the changes have a look and let me know if this time it works.